### DISCUSSION TITLE
Recent change to createSocketURL.js causing error in IE11

### DISCUSSION THREAD

**Discussion by akolssontd** [webpack/webpack#15700] [Q&A]
I'm getting the following error in IE11: Object doesn't support property or method 'endsWith'

It appears to be caused by a recent (3/18/2022) change in createSocketURL.js where 
```
if (protocol && protocol.substr(-1) !== ":") {
    protocol += ":";
}
```
was changed to
```
if (protocol && !protocol.endsWith(":")) {
    protocol += ":";
}
```

I've tried to add a polyfill for "endsWith", but it doesn't appear to work. Perhaps I'm not adding it in the right place (index.js of my React application).

Does anyone know where I should be adding the polyfill or is there a way to prevent the code in question from being included in my dev build?

**Comment by vankop** [comment]
@alexander-akait

**Reply by alexander-akait** [reply]
hm, I see, I will fix today

**Reply by alexander-akait** [reply]
https://github.com/webpack/webpack-dev-server/pull/4403
